### PR TITLE
Recipe book

### PR DESCRIPTION
**Pull request by Emdorkable**
new firebase, recipe book

### CODE REVIEW

**Comment by willeng37** [comment]
I think the only issues that probably need to be checked are rendering errors due to a null (non-logged in) user for any protected pages.

**Review by danthynguyen — DISMISSED** [review]
the recipes are not rendering on the recipe book page

there is an uncaught promise in RecipeBook.js 44:1, looks like its trying to convert an undefined or null into an object

image

I think there might be something wrong with the fetch in the function

**Review by danthynguyen — APPROVED** [review]
list of issues to address later
- only renders on refresh on first time
- when reaccessing the recipe book, takes 5 seconds to render, otherwise looking at a blank page
- memory-wasting on firebase

File: src/components/NewRecipe.js [hunk 1]
```diff
@@ -137,8 +142,9 @@ export default function NewRecipe() {
     function Finalize() {
         return (
             <   DisplayRecipeSummary
-                recipeIndex={currentRecipeIndex}
-                currentIngredients={currentIngredients} />
+            currentIngredients={currentIngredients} 
+            changeAddedIngredients={changeAddedIngredients}
```
Comment (jcderry5) [inline src/components/NewRecipe.js:142]: remove changeAddedIngredients :)

**Review by jcderry5 — APPROVED** [review]
LGTM

**Review by jcderry5 — APPROVED** [review]
Trusting Will

**Pull request merged**